SPECS/luajit/luajit.spec

```diff
@@ -0,0 +1,261 @@
+%global rctag beta3
+
+Summary:        Just-In-Time Compiler for Lua
+Name:           luajit
+Version:        2.1.0
+%global apiver %(v=%{version}; echo ${v%.${v#[0-9].[0-9].}})
+%global srcver %{version}%{?rctag:-%{rctag}}
```

Comment:
NIT:
Should we move them macros to the top as per our convention.

Reply:
Good to keep here since macros using %{version} value. Hence they were listed below Version tag.